components/image/drawing/drawing.p5m

```diff
@@ -22,7 +22,13 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
+# python apps are unusable without python runtime binary
+depend type=require fmri=__TBD pkg.debug.depend.file=python$(PYVER) \
```

Comment:
Wouldn't it be simpler to adjust the `pkg.depend.bypass-generate` transform below to do not complain about missing `drawing/__init__.py` and leave `pkgdepend` to do its job instead?

Reply:
I would love for gmake REQUIRED_PACKAGES auto detect python modules and add those REQUIRED_PACKAGES to the Makefile.  I have zero insight how to make it work for this package.

If I replace the line `<transform file path=usr/share/drawing/drawing/.*\.py$ -> add pkg.depend.bypass-generate .* >` with ```
<transform file path=usr/share/drawing/drawing/.*\.py$ -> add pkg.depend.bypass-
generate .* >
``` then gmake REQUIRED_PACKAGES completes without an error and the results of the Makefile does not change.  Maybe I do not understand what you are trying to say?  Pointers to an existing .p5m file that does what you suggest will help to solve this problem.

Reply:
> If I replace the line `<transform file path=usr/share/drawing/drawing/.*\.py$ -> add pkg.depend.bypass-generate .* >` with `<transform file path=usr/share/drawing/drawing/.*\.py$ -> add pkg.depend.bypass- generate .* >`

Could you please explain what do you want to achieve by the added space to the transform?  Maybe I miss something important but the only difference in your _"If I replace the line X with Y"_ sentence above between the _X_ and _Y_ seems to be literally the single added space only.

Anyway, my point is this:

I did this:
```
$ git diff drawing.p5m 
diff --git a/components/image/drawing/drawing.p5m b/components/image/drawing/drawing.p5m
index bfbe90b589..58db09d7ae 100644
--- a/components/image/drawing/drawing.p5m
+++ b/components/image/drawing/drawing.p5m
@@ -22,7 +22,7 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
-<transform file path=usr/bin/drawing -> add pkg.depend.bypass-generate .* >
+<transform file path=usr/bin/drawing -> add pkg.depend.bypass-generate .*drawing.* >
 
 # mate menu icon needs to be in /usr/share/pixmaps
 file usr/share/icons/hicolor/128x128/apps/com.github.maoschanz.drawing.png path=usr/share/pixmaps/com.github.maoschanz.drawing.png
$
```
and everything seems to work as expected.

So my question is: why to add several lines here to somehow achieve what can be done by simple online change?